telegram/chat.py

```diff
@@ -150,6 +150,20 @@ def link(self) -> Optional[str]:
             return "https://t.me/{}".format(self.username)
         return None
 
+    @property
+    def is_anonymous_admin(self) -> bool:
+        """:obj:`bool`: :obj:`True`, if this chat is with is the bot representing anonymous admins.
+            This behaviour is undocumented and might be changed by Telegram. """
```

Comment:
maybe call it convenience property? We are doing it like that in eg [user.py](https://github.com/python-telegram-bot/python-telegram-bot/blob/264b2c9c72691c5937b80e84e061c52dd2d8861a/telegram/user.py#L101)